ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1485,3 +1485,33 @@
                 timeout: 135
                 type: absolute
             publishers: []
+      - '{name}-{test_name}-for-pull-request':
+            test_name: kind-flexible-ipam-e2e
+            node: 'antrea-kind-flexible-ipam-testbed'
+            description: 'This is the {test_name} test for {name}.'
+            branches:
+            - ${{sha1}}      
+            builders:
+              - builder-kind-flexible-ipam-e2e:
+                  kind_cluster_name: '{test_name}'
+            trigger_phrase: ^(?!Thanks for your PR).*/test-kind-flexible-ipam-e2e.*
+            white_list_target_branches: []
+            allow_whitelist_orgs_as_admins: true
+            admin_list: '{antrea_admin_list}'
+            org_list: '{antrea_org_list}'
+            white_list: '{antrea_white_list}'
+            only_trigger_phrase: true
+            trigger_permit_all: true
+            status_context: jenkins-kind-flexible-ipam-e2e
+            status_url: --none--
+            success_status: Build finished.
```

Comment:
@KMAnju-2021 I see the job has succeeded on Jenkins, but the result has not been received by GitHub. Could you double-check if this was caused by a Jenkins configuration error?

Reply:
done